transportable-udfs-spark_2.11/src/test/scala/com/linkedin/transport/spark/data/TestSparkArray.scala

```diff
@@ -15,41 +15,41 @@ import org.testng.annotations.Test
 
 class TestSparkArray {
 
-  val stdFactory = new SparkTypeFactory(null)
+  val typeFactory = new SparkTypeFactory(null)
   val arrayData = ArrayData.toArrayData(Array.range(0, 5)) // scalastyle:ignore magic.number
   val arrayType = ArrayType(DataTypes.IntegerType)
 
   @Test
   def testCreateSparkArray(): Unit = {
-    val stdArray = SparkConverters.toTransportData(arrayData, arrayType).asInstanceOf[data.ArrayData[Integer]]
-    assertEquals(stdArray.size(), arrayData.numElements())
-    assertSame(stdArray.asInstanceOf[PlatformData].getUnderlyingData, arrayData)
+    val array = SparkConverters.toTransportData(arrayData, arrayType).asInstanceOf[data.ArrayData[Integer]]
+    assertEquals(array.size(), arrayData.numElements())
+    assertSame(array.asInstanceOf[PlatformData].getUnderlyingData, arrayData)
   }
 
   @Test
   def testSparkArrayGet(): Unit = {
-    val stdArray = SparkConverters.toTransportData(arrayData, arrayType).asInstanceOf[data.ArrayData[Integer]]
-    (0 until stdArray.size).foreach(idx => {
-      assertEquals(stdArray.get(idx), idx)
+    val row = SparkConverters.toTransportData(arrayData, arrayType).asInstanceOf[data.ArrayData[Integer]]
+    (0 until row.size).foreach(idx => {
+      assertEquals(row.get(idx), idx)
     })
   }
 
   @Test
   def testSparkArrayAdd(): Unit = {
-    val stdArray = SparkConverters.toTransportData(arrayData, arrayType).asInstanceOf[data.ArrayData[Integer]]
-    stdArray.add(5)
+    val row = SparkConverters.toTransportData(arrayData, arrayType).asInstanceOf[data.ArrayData[Integer]]
+    row.add(5)
     // Since original ArrayData is immutable, a mutable ArrayBuffer should be created and set as the underlying object
-    assertNotSame(stdArray.asInstanceOf[PlatformData].getUnderlyingData, arrayData)
-    assertEquals(stdArray.size(), arrayData.numElements() + 1)
-    assertEquals(stdArray.get(stdArray.size() - 1), 5)
+    assertNotSame(row.asInstanceOf[PlatformData].getUnderlyingData, arrayData)
+    assertEquals(row.size(), arrayData.numElements() + 1)
+    assertEquals(row.get(row.size() - 1), 5)
   }
 
   @Test
   def testSparkArrayMutabilityReset(): Unit = {
-    val stdArray = SparkConverters.toTransportData(arrayData, arrayType).asInstanceOf[data.ArrayData[Integer]]
-    stdArray.add(5)
-    stdArray.asInstanceOf[PlatformData].setUnderlyingData(arrayData)
+    val row = SparkConverters.toTransportData(arrayData, arrayType).asInstanceOf[data.ArrayData[Integer]]
```

Comment:
Here and in other places you are renaming an array to a row.

Reply:
addressing

Reply:
addressed